### DISCUSSION TITLE
Elastalert2 and Elasticsearch 6.4.1

### DISCUSSION THREAD

**Discussion by dcl4k** [jertel/elastalert2#1509] [Q&A] [answered]
(First I'd like to say that 'yes' I am aware how old 6.4.1 is)

Good morning, I've recently been asked to look into using Elastalert with our Elasticsearch cluster. I have tried Elastalert and Elastalert2. Elastalert2 simply returns an error telling me the Elasticsearch version is not supported, and Elastalert returns errors about the doctype. 

My preference would be to use Elastalert2 since Elastalert is deprecated. Is there any plan to support older versions of Elasticsearch? 

If this is simply out of the question no problem, thanks for taking the time to respond to read this.

**Comment by nsano-rururu** [comment, marked as answer]
Support for ElasticSearch 6 has been discontinued in 2.4.0. Please use a version earlier than 2.4.0.
https://github.com/jertel/elastalert2/releases/tag/2.4.0

**Reply by dcl4k** [reply]
Thanks!

**Comment by wzglc6** [comment]
Good afternoon,
we are still on Elasticsearch 5.6.16, what version would support that ancient version?

Thank you.

**Reply by nsano-rururu** [reply]
The Elasticsearch 5 code has also been removed in support of Elasticsearch 8. Therefore, if you use Elasticsearch 5, you will need to use a version of elastalert earlier than 2.4.0, just like Elasticsearch 6.
